README.md

```diff
@@ -3,9 +3,9 @@
 Slack API Client for Deno Run on Slack projects
 
 ```ts
-import { SlackAPI } from "https://deno.land/x/deno_slack_api@0.0.5/mod.ts"
+import { SlackAPI } from "https://deno.land/x/deno_slack_api/mod.ts"
```

Comment:
I think we do want to keep the version specifier, even in the README, as _someone_ will just copy-paste the example into their app and before they know it the app is in production, we release a breaking change, and they will have a bad time.

I know that's annoying, as then we have to remember to update that on every release, but I think that's a price we must pay at this point.